haikuwebkit-1.5.1.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Open source web browser engine"
+DESCRIPTION="WebKit is an open source web browser engine. WebKit is also the name of the \
+Mac OS X system framework version of the engine that's used by Safari, \
+Dashboard, Mail, and many other OS X applications. WebKit's HTML and \
+JavaScript code began as a branch of the KHTML and KJS libraries from KDE."
+HOMEPAGE="http://www.webkit.org/"
+COPYRIGHT="1998-2015 Apple Inc., Google Inc., et al"
+LICENSE="
+	GNU LGPL v2
+	GNU LGPL v2.1
+	MIT
+	"
+REVISION="1"
+SOURCE_URI="https://github.com/haiku/webkit/archive/$portVersion.tar.gz"
+SOURCE_FILENAME="haikuwebkit-$portVersion.tar.gz"
+CHECKSUM_SHA256="f434e707ba958412d073d1ca2010e934281662cf88b4cc9a6b2735fe2d96fd92"
+SOURCE_DIR="webkit-$portVersion"
+PATCHES="haikuwebkit-1.5.1.patchset"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	haikuwebkit$secondaryArchSuffix = $portVersion
+	lib:libWebKit$secondaryArchSuffix = $portVersion
+	lib:libJavaScriptCore$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:jsc
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libicuuc$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libwebp$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	haikuwebkit${secondaryArchSuffix}_devel = $portVersion
+	devel:libJavaScriptCore$secondaryArchSuffix = $portVersion
+	devel:libwebcore$secondaryArchSuffix = $portVersion
+	devel:libWebKit$secondaryArchSuffix = $portVersion
+	devel:libwtf$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	haikuwebkit$secondaryArchSuffix == $portVersion base
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	haiku_devel
+	devel:libicuuc$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:pkg_config
+	cmd:flex
+	cmd:bison
+	cmd:gcc$secondaryArchSuffix
+	cmd:gperf
+	cmd:ld$secondaryArchSuffix
+	cmd:m4
+	cmd:make
+	cmd:ninja
+	cmd:perl
+	cmd:python
+	cmd:cmake >= 3.0.0
+	cmd:ruby
+	cmd:which
+	"
```

Comment:
You don't have enough whitespace between sections, and some of this is out of order. See https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering for the proper ordering and where blank lines should be between sections.